bin/utils/update_browsers.js

```diff
@@ -0,0 +1,75 @@
+/* global __dirname */
+
+const fs = require('fs');
+const path = require('path');
+const browserslist = require('browserslist');
+const semver = require('semver');
+const diff = require('diff');
+const chalk = require('chalk');
+
+(() => {
```

Comment:
is there any need for this IIFE? AFAIK strict mode can also be enabled for the whole file

Reply:
If the "node env" is not on, ESLint requires that you use the "function form" of strict mode.
But since I'm turning "node env" on, I'll be able to use the global form.